parcel/Example_P3_vs_activitybased.jl

```diff
@@ -0,0 +1,217 @@
+        MK.lines!(ax3, sol.t, sol[3, :], color = :blue)                                        # Temperature
+    elseif deposition == "ABDINM"
+        for aerosol in aerosol_deposition
+            local params = parcel_params{FT}(
+                const_dt = const_dt,
+                r_nuc = r_nuc,
+                w = w,
+                aerosol = aerosol,
+                deposition = deposition,
+                deposition_growth = deposition_growth,
+                size_distribution = size_distribution,
+            )
+            # solve ODE
+            local sol = run_parcel(IC_dep, FT(0), t_max, params)
+            # Plot
+            if aerosol == CMP.Feldspar(FT)
+                line_color = :green
+            elseif aerosol == CMP.Ferrihydrite(FT)
+                line_color = :orange
+            elseif aerosol == CMP.Kaolinite(FT)
+                line_color = :magenta
+            end
+            MK.lines!(ax1, sol.t, S_i.(sol[3, :], (sol[1, :])), label = deposition, color = line_color) # saturation
+            MK.lines!(ax2, sol.t, sol[9, :] * 1e-6, color = line_color)                                 # ICNC
+            MK.lines!(ax3, sol.t, sol[3, :], color = line_color)                                        # Temperature
+        end
+    end
+end
+
+for heterogeneous in heterogeneous_modes
+    if heterogeneous == "P3_het"
+        local params = parcel_params{FT}(
+            const_dt = const_dt,
+            r_nuc = r_nuc,
+            w = w,
+            aerosol = aerosol_none,
+            heterogeneous = heterogeneous,
+            deposition_growth = deposition_growth,
+            size_distribution = size_distribution,
+        )
+        # solve ODE
+        local sol = run_parcel(IC_het, FT(0), t_max, params)
+        # Plot
+        MK.lines!(ax4, sol.t, S_i.(sol[3, :], (sol[1, :])), label = heterogeneous, color = :blue) # saturation
+        MK.lines!(ax5, sol.t, sol[9, :] .* 1e-6, color = :blue)                                   # ICNC
+        MK.lines!(ax6, sol.t, sol[3, :], color = :blue)                                           # Temperature
+    elseif heterogeneous == "ABIFM"
+        for aerosol in aerosol_heterogeneous
+            local params = parcel_params{FT}(
+                const_dt = const_dt,
+                r_nuc = r_nuc,
+                w = w,
+                aerosol = aerosol,
+                heterogeneous = heterogeneous,
+                deposition_growth = deposition_growth,
+                size_distribution = size_distribution,
+            )
+            # solve ODE
+            local sol = run_parcel(IC_het, FT(0), t_max, params)
+            # Plot
+            if aerosol == CMP.DesertDust(FT)
+                line_color = :green
+                MK.lines!(ax4, sol.t, S_i.(sol[3, :], (sol[1, :])), label = heterogeneous, color = line_color) # saturation
+                MK.lines!(ax5, sol.t, sol[9, :] .* 1e-8, color = line_color)                                   # ICNC 
```

Comment:
I think it's better to keep `1e-6` everywhere and switch to the log scale. Or keep the scale but just mention in the text that describes the plot that the values from this configuration are not zero by 2 orders of magnitude smaller than the rest.